community/clinics/NMRA2008/index.shtml

```diff
@@ -16,8 +16,9 @@
 
 <hr class="hide">
 
-<div class="no-sidebar" id="mBody">
-	<div id="mainContent">
+<div id="mBody">
+    <!--#include virtual="../Sidebar.shtml" -->
+    <div id="mainContent">
```

Comment:
Should line 20 be referencing SidebarCommunity.shtml?

Reply:
../Sidebar contains an include for that, plus one for clinic details. So I don't think so